ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -22,6 +23,7 @@ const DEFAULT_MAX_DELETE_COUNT = 2;
 const DEFAULT_MAX_RESUME_COUNT = 2;
 // We allow a certain # of server errors to occur before we error-out of the initialization flow.
 const DEFAULT_MAX_SERVER_ERROR_COUNT = 10;
+const DEFAULT_RUNTIME_CONFIG = runtimePresets.generalAnalysis.runtimeTemplate;
```

Comment:
nit: ideally the default would follow this logic, but if that requires a lot of code changes you can just leave it as is (GCE will be launched soon, so we probably won't need to worry about this):

https://github.com/all-of-us/workbench/blob/f272563b7256f6e10d9acc537b7f605f65dee368/ui/src/app/utils/leo-runtime-initializer.tsx#L194-L198

Reply:
Don't bother with this anymore, I don't think, since I've already started enabling GCE by default. The way you have it now should work fine.